### DISCUSSION TITLE
Order of losses

### DISCUSSION THREAD

**Discussion by shanmukhi1998** [lululxvi/deepxde#1133] [General]
Hi.
Can anyone please clarify what is the order of losses that are displayed after every set of iterations?
Step      Train loss              Test loss               Test metric
0         [2.63e+12, 6.19e-02]    [2.64e+12, 6.19e-02]    []  
1000      [4.98e+07, 5.10e-02]    [4.20e+07, 5.10e-02]    []

In the above case, the first one is PDE loss or the second one?

Any help appreciated. Thanks.

**Comment by lululxvi** [comment]
See FAQ Q: What is the output of DeepXDE? How can I visualize the results?

**Reply by tangri** [reply]
Hi @lululxvi ,

Looking through the FAQ Q, referenced, I found info on train.dat but not loss.dat. Could you explain the contents of loss.dat? I noticed that the number of elements in loss.dat is the same as the number of equations plus the number of IC_BC

Also how to the write a loss.dat while the training is going on and has not yet finished?
